### PR TITLE
Add Python 3.6, drop Python 3.3, 100% coverage.

### LINKED ISSUE #4: Add Python 3.6, drop Python 3.3

### PR DESCRIPTION
**Pull request by jamadden**
Fixes #4.

Currently based on #5 (the first commit).

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -1,16 +1,20 @@
...
 
+- Host documentation at https://zopeprincipalregistry.readthedocs.io
+
+- Reach 100% test coverage and ensure we remain there.
 
 4.0.0 (2014-12-24)
-------------------
+==================
 
 - Add support for PyPy.  (PyPy3 is pending release of a fix for:
```
Comment (icemac) [inline CHANGES.rst]: As PyPy3 is now tested via TravisCI it could be mentioned in version 4.1 that it is supported now, too.

Comment (jamadden) [reply CHANGES.rst]: OK, will do.

**Comment by jamadden** [comment]
Thanks! Can we get a PyPI release when convenient, or access to do so?

**Comment by icemac** [comment]
@jamadden I added you as owner of this package on PyPI. (Sorry for the delay.)

**Comment by jamadden** [comment]
No worries, and thank you! 4.1.0 has been released to PyPI.

**Pull request merged**